x-pack/plugins/security_solution/server/lib/detection_engine/rule_preview/api/preview_rules/route.ts

```diff
@@ -263,6 +263,7 @@ export const previewRulesRoute = async (
               startedAt: startedAt.toDate(),
               state: statePreview,
               logger,
+              flappingSettings: DEFAULT_FLAPPING_SETTINGS,
```

Comment:
I think we should create `DISABLE_FLAPPING_SETTINGS` to disable the flapping for security solution since they are not using the life cycle alert, It will be less confusing it in the long run.

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/149448/commits/39c3d93b63abab6d910c0cc39d201b296d9d83a2